src/transform3d.rs

```diff
@@ -633,6 +633,23 @@ where T: Copy + fmt::Debug +
     }
 }
 
+impl<T: Copy, Src, Dst> Into<[T; 16]> for TypedTransform3D<T, Src, Dst> {
```

Comment:
don't you see it somewhat inconsistent? it's almost asking for `AffineTransform3D` being different from just `Transform3D`.